src/components/UseOfFunds/Icons/IconClose.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+	<svg width="32" height="32" viewBox="0 0 32 32" fill="#ffffff" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This needs to respect dark mode so we remove the fill and make sure it has a background in css using my suggestions in the main comment:

```suggestion
	<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```